Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -594,6 +611,11 @@ var NavigatorIOS = React.createClass({
     var shouldUpdateChild = this.state.updatingAllIndicesAtOrBeyond !== null &&
       this.state.updatingAllIndicesAtOrBeyond >= i;
 
+    var titleStyles = precomputeStyle(flattenStyle(this.props.titleStyle));
+    if (titleStyles == null) {
```

Comment:
This should be a check for `!titleStyles` since people sometimes pass in false for styles